UnitTests/Onboarding/ContextualOnboarding/ContextualOnboardingStateMachineTests.swift

```diff
@@ -315,22 +320,340 @@ class ContextualOnboardingStateMachineTests: XCTestCase {
         }
     }
 
+    func test_UpdateStateFor_OpensDuckDuckGoSearch_ShowsTryASearch() {
```

Comment:
Should we keep the naming of these tests consistent with the rest of the tests already written? 

e.g. test_UpdateStateFor_When<condition>_Then<whathappens>